src/mailing_info.cpp

```diff
@@ -58,8 +58,6 @@ auto lwg::make_html_anchor(lwg::issue const & iss) -> std::string {
    auto temp = std::to_string(iss.num);
 
    std::string result{"<a href=\""};
-   result += filename_for_status(iss.stat);
-   result += '#';
```

Comment:
This part changes `<iref>` links to point to `1234` instead of `lwg_defects.html#1234`. GitHub pages allow you to omit the trailing `.html` suffix in the URL.